diff --git a/runcoms/zpreztorc b/runcoms/zpreztorc
index a549073..0cc1d60 100755
--- a/runcoms/zpreztorc
+++ b/runcoms/zpreztorc
@@ -35,6 +35,7 @@ zstyle ':prezto:load' pmodule \
   'spectrum' \
   'utility' \
   'completion' \
+  'git' \
   'prompt'
 
 #
@@ -113,7 +114,7 @@ zstyle ':prezto:module:editor' key-bindings 'emacs'
 # Set the prompt theme to load.
 # Setting it to 'random' loads a random theme.
 # Auto set to 'off' on dumb terminals.
-zstyle ':prezto:module:prompt' theme 'sorin'
+zstyle ':prezto:module:prompt' theme 'pure'
 
 # Set the working directory prompt display length.
 # By default, it is set to 'short'. Set it to 'long' (without '~' expansion)
diff --git a/runcoms/zshrc b/runcoms/zshrc
deleted file mode 100755
index 039b882..0000000
--- a/runcoms/zshrc
+++ /dev/null
@@ -1,13 +0,0 @@
-#
-# Executes commands at the start of an interactive session.
-#
-# Authors:
-#   Sorin Ionescu <sorin.ionescu@gmail.com>
-#
-
-# Source Prezto.
-if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
-  source "${ZDOTDIR:-$HOME}/.zprezto/init.zsh"
-fi
-
-# Customize to your needs...
diff --git a/runcoms/zshrc b/runcoms/zshrc
new file mode 120000
index 0000000..3fd4b15
--- /dev/null
+++ b/runcoms/zshrc
@@ -0,0 +1 @@
+/home/nkaviratna/.dotfiles/zshrc
\ No newline at end of file
